src/styles/core/colors/_light.scss

```diff
@@ -176,6 +176,7 @@
   --color-tabnav-item-border-color: var(--color-fill-gray);
   --color-text: var(--color-figure-gray);
   --color-text-background: var(--color-fill);
+  --color-other-decl-button: var(--color-text-background);
```

Comment:
Super tiny thing, but most* of these are sorted alphabetically, and it would be nice to keep that pattern going so it's easier to scan through.

Reply:
Makes sense, although I don't think that's the case since the value gets computed at runtime, so the source order shouldn't matter—test it out and double check me though!